__tests__/navigations.spec.ts

```diff
@@ -198,7 +198,7 @@ describe('Navigations', () => {
     expect(wrapper.text()).toBe('/bar')
   })
 
-  it.skip('can wait for an ongoing navigation', async () => {
+  it('can wait for an ongoing navigation', async () => {
```

Comment:
I took the liberty to unskip this test, as it was passing. Let me know if that needs to be reverted.